quantumCoinFlipping/1_Honest_Participants/aliceCoin.py

```diff
@@ -0,0 +1,82 @@
+from SimulaQron.general.hostConfig import *
+from SimulaQron.cqc.backend.cqcHeader import *
+from SimulaQron.cqc.pythonLib.cqc import *
+
+import random
+import numpy
```

Comment:
I would suggest using json files / json library for config instead of loading numpy.